src/main/java/core/basesyntax/service/impl/FruitTransactionImpl.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FruitTransaction;
+import core.basesyntax.strategy.FruitOperation;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class FruitTransactionImpl implements FruitTransaction {
+    private static final int INDEX_OPERATION = 0;
+    private static final int INDEX_FRUIT = 1;
+    private static final int INDEX_AMOUNT = 2;
+    private static final int DEFAULT = 0;
```

Comment:
remove this constant

Reply:
What about this comment? You don't use DEFAULT anywhere

Reply:
I had 5 comments collapsed, I didn't see